lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -95,6 +100,14 @@
         resource_group: Testing
         disk_size_gb: 4
 
+    - name: Create managed operating system disk from page blob
+      azure_rm_managed_disk:
+        name: mymanageddisk
```

Comment:
there's resource group missing in that sample

Reply:
I added the missing fields.